tests/pyccel/test_pyccel.py

```diff
@@ -46,7 +46,7 @@ def get_exe(filename, language=None):
 def insert_pyccel_folder(abs_path):
     base_dir = os.path.dirname(abs_path)
     base_name = os.path.basename(abs_path)
-    return os.path.join(base_dir, "__pyccel__", base_name)
+    return os.path.join(base_dir, "__pyccel__"+os.environ.get('PYTEST_XDIST_WORKER', ''), base_name)
```

Comment:
Could you add some empty spaces around the `+` binary operator?

Reply:
Fixed b245ece3

Reply:
If you care about these spaces, should we have a pep8 linter?

Reply:
I do not know. The PEP8 might be too aggressive when we align things vertically for readability.